plugin/trino-faker/src/test/java/io/trino/plugin/faker/TestFakerQueries.java

```diff
@@ -495,7 +495,14 @@ void testCreateTableAsSelect()
 
         for (TestDataType testCase : testCases) {
             try (TestTable sourceTable = new TestTable(getQueryRunner()::execute, "ctas_src_" + testCase.name(), "(%s) WITH (null_probability = 0, default_limit = 1000)".formatted(testCase.columnSchema()));
-                    TestTable table = new TestTable(getQueryRunner()::execute, "ctas_" + testCase.name(), "WITH (null_probability = 0, default_limit = 1000) AS SELECT %s FROM %s".formatted(testCase.name(), sourceTable.getName()))) {
+                    TestTable table = new TestTable(getQueryRunner()::execute, "ctas_" + testCase.name(), "WITH (null_probability = 0, default_limit = 1000, max_dictionary_size = 0) AS SELECT %s FROM %s".formatted(testCase.name(), sourceTable.getName()))) {
```

Comment:
is there some coverage added?

Reply:
No, but it's required to keep the number of distinct values high enough.